experimental/sdg-tracker/src/components/countries/CountriesContent.tsx

```diff
@@ -512,10 +512,7 @@ const ChartGoalBlock: React.FC<{
   return (
     <>
       {placeDcids[0] === EARTH_PLACE_DCID && (
-        <GoalOverview
-          goalNumber={Number(goal)}
-          showExploreLink={false}
-        />
+        <GoalOverview goalNumber={Number(goal)} showExploreLink={false} />
```

Comment:
meant to be false?

Reply:
I think this is a different explore link, not related to the chart tile. I will leave it untouched.

Reply:
sg

Reply:
False is correct. it should only be true if we're on the "all goals for the world" page.